projects/UIP.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: project
+title: UIP
+permalink: projects/UIP
+description: A cross-platfrom desktop application for wallpapers.
```

Comment:
But that makes the string look rather too long, and anyway we do have a `What is UIP?` section that explains it, right?